sway-lsp/src/server.rs

```diff
@@ -291,25 +291,25 @@ mod tests {
     use tower_lsp::jsonrpc::{self, Request, Response};
     use tower_lsp::LspService;
 
-    fn _e2e_test_dir() -> PathBuf {
+    fn e2e_test_dir() -> PathBuf {
         env::current_dir()
             .unwrap()
             .parent()
             .unwrap()
             .join("test/src/e2e_vm_tests/test_programs/should_pass/language")
-            .join("enum_if_let")
+            .join("struct_field_access")
     }
 
-    fn sway_example_dir() -> PathBuf {
+    fn _sway_example_dir() -> PathBuf {
```

Comment:
what's this for?

Reply:
I have 2 helper methods for testing against the sway examples folder and another for the e2e language folder. Clippy complains if there are any unused functions so I underscore the one I'm not currently using for development. 

This is actually one of the main motivations for me wanting to inspect the syntax tree in the editor for the currently open file in VScode.

Reply:
Ah ok, fair. what about just doing `#[allow(dead_code)]` on both?

Reply:
Yeah good idea. I'll add that to my next PR. Cheers.